lib/puppet/indirector/report/rest_spec.rb

```diff
@@ -31,7 +31,7 @@ def save(request)
     # Generate serverspec files
     # TODO: Check that we get the catalog from cache
     resources = Puppet::Resource::Catalog.indirection.find(request.instance.host).resources
-    spec_dir = File.join(Puppet[:vardir], 'policy', 'server')
+    spec_dir = File.join(Puppet[:vardir], 'lib', 'policy', 'server')
```

Comment:
You are right that this is wrong. However, the solution is using `:libdir` instead of `:vardir`. If you fix this, I'll happily merge the change.